backend/tournesol/lib/public_dataset.py

```diff
@@ -264,9 +264,8 @@ def write_metadata_file(write_target, data_until: Optional[datetime] = None) ->
                 "TRUSTED_EMAIL_PRETRUST": TRUSTED_EMAIL_PRETRUST,
                 "VOUCH_DECAY": VOUCH_DECAY,
             },
+            "individual_scores": individual_scores_algo.get_metadata(),
```

Comment:
Small inconsistency: We don't guarantee that the scores computed are indeed computed with this algo. The worse edgecase would be if we deploy a new algorithm on Sunday evening right before the re-computation of the dataset

Reply:
Indeed. We could consider saving the parameters in the database. But the situation could become more complex with online updates where scores for different users / entities will be computed at different times with potentially different configurations of the algorithm.

This was briefly discussed when implementing https://github.com/tournesol-app/tournesol/pull/1418